src/applications/simple-forms/21-10210/definitions/form-address.js

```diff
@@ -0,0 +1,327 @@
+// copy of src/platform/forms/definitions/address.js,
```

Comment:
which part of this file has new changes compared to the common address definition? i'm especially interested because I'll be working on address pattern soon.  Also, is it possible , whatever change was made, to make to the original? or we really needed a full duplication?

Reply:
I'll follow-up w/ a original-definition refactor after finishing this form.  Pat needs this PR merged ASAP.